adbc.h

```diff
@@ -1061,7 +1048,7 @@ typedef AdbcStatusCode (*AdbcDriverInitFunc)(size_t count, struct AdbcDriver* dr
                                              struct AdbcError* error);
 
 // For use with count
-#define ADBC_VERSION_0_0_1 27
+#define ADBC_VERSION_0_0_1 26
```

Comment:
Is this intentional?

Reply:
I think we will want to avoid changing a signature in either case to avoid causing ABI/compatibility issues.

I will write this up more clearly in #96. I think I'll sketch out the unique number approach because I think it is easier to understand, as well, and if so I'll switch things in that PR. 

Thanks!